src/aiodynamo/client.py

```diff
@@ -1,70 +1,76 @@
-from enum import Enum, unique
-from functools import partial
-from itertools import chain
-from typing import Any, AsyncIterator, Dict, List, TypeVar, Union
-
-import attr
-from boto3.dynamodb.conditions import ConditionBase, ConditionExpressionBuilder
-from boto3.dynamodb.types import DYNAMODB_CONTEXT
-from botocore.exceptions import ClientError
-
-from .errors import EmptyItem, ItemNotFound, TableNotFound
-from .models import (
+from __future__ import annotations
+
+import asyncio
+import datetime
+from dataclasses import dataclass
+from typing import *
+
+from aiodynamo.errors import *
+from aiodynamo.models import (
     GlobalSecondaryIndex,
     KeySchema,
     KeySpec,
     KeyType,
     LocalSecondaryIndex,
-    ProjectionExpr,
     ReturnValues,
     Select,
     StreamSpecification,
     TableDescription,
     TableStatus,
+    ThrottleConfig,
```

Comment:
I wonder if perhaps these dynamo things should be re-exported at the top-level,
so that users of the library could do `xx = aiodynamo.ThrottleConfig(...)` 🤔

Reply:
I like having the namespaces.